api/models/userMethods.js

```diff
@@ -71,6 +73,16 @@ const createUser = async (data, disableTTL = true, returnUser = false) => {
   }
 
   const user = await User.create(userData);
+
+  if (isEnabled(process.env.CHECK_BALANCE) && process.env.START_BALANCE) {
```

Comment:
This change requires a documentation update in `.env.example` and:
- https://www.librechat.ai/docs/configuration/dotenv#balance
- https://www.librechat.ai/docs/configuration/token_usage

Reply:
Information about START_BALANCE has been added to the .env file

Reply:
Information about START_BALANCE has been added to https://github.com/LibreChat-AI/librechat.ai/pull/156